controller/crio-runtime/runtime.go

```diff
@@ -0,0 +1,191 @@
+// /*
+// Copyright 2020 Kaloom Inc.
+// Copyright 2014 The Kubernetes Authors.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// */
+
+package crioruntime
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"k8s.io/kubernetes/pkg/kubelet/util"
+)
+
+const (
+	crioNetNSFmt = "/run/netns/%s"
```

Comment:
The NS folder is under `/var/run/netns`, although `/var/run` is a symbolic link to `/run` we should not assume it's the case on all system; therefore:

crioNetNSFmt = "/var/run/netns/%s" is more appropriate

Reply:
Fixed in fps as well